nessie/src/test/java/org/apache/iceberg/nessie/TestCustomNessieClient.java

```diff
@@ -78,30 +77,11 @@ public void testNonExistentCustomClient() {
                       temp.toUri().toString(),
                       CatalogProperties.URI,
                       uri,
-                      NessieConfigConstants.CONF_NESSIE_CLIENT_BUILDER_IMPL,
-                      "non.existent.ClientBuilderImpl"));
-            })
-        .isInstanceOf(RuntimeException.class)
-        .hasMessageContaining("Cannot load Nessie client builder implementation class");
-  }
-
-  @Test
-  public void testCustomClientByImpl() {
```

Comment:
Why this is removed? I think we still support the custom client. Just the property name changed?

Reply:
we do have the same test case [here](https://github.com/apache/iceberg/blob/b5ea0d5a7f55e5b8d9eec8e764bbcc35f8301db3/nessie/src/test/java/org/apache/iceberg/nessie/TestCustomNessieClient.java#L108) .

Reply:
ACK

Reply:
These tests are different, this one uses `NessieConfigConstants.CONF_NESSIE_CLIENT_BUILDER_IMPL`, the other one uses `NessieConfigConstants.CONF_NESSIE_CLIENT_NAME`, right?

Reply:
Since CONF_NESSIE_CLIENT_BUILDER_IMPL is deprecated, we need change testcase to use CONF_NESSIE_CLIENT_NAME instead of removing. But there is already a testcase to do that. So, I guess he removed the testcase of CONF_NESSIE_CLIENT_BUILDER_IMPL

Reply:
I see. I missed the deprecation. The change makes sense to me.